Event/Events.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the ACME PHP library.
+ *
+ * (c) Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AcmePhp\Bundle\Event;
+
+/**
+ * Contains all events thrown in the AcmePhp bundle.
+ *
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+final class Events
```

Comment:
I usually name this `AcmePhpBundleEvents` and put it on bundle root, to make a parallel with the bundle class `AcmePhpBundle`. What do you think?